recipes/erdantic/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "erdantic" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/erdantic-{{ version }}.tar.gz
+  sha256: 96a9bef0cc8ed5ae34411af5aba242ba3dac679a14bb19379e89c0d41dc72492
+
+build:
+  number: 0
+  skip: true   # [py2k or win]
```

This looks like it could be made noarch as `dataclasses` and `importlib-metadata` are available as metapackages for newer Python versions so the selectors could be removed.
```suggestion
  noarch: python
```